tests/UnitTests/ApiConnectors/BankTransactionApiConnectorTest.php

```diff
@@ -195,4 +196,33 @@ public function testSendAllReturnsMappedObjectsAllLineTypes()
         $this->assertEquals(Money::EUR(10000), $line->getVatTurnover());
         $this->assertEquals(Money::EUR(10000), $line->getVatBaseTurnover());
     }
+
+    /**
+     * @expectedException Exception
+     * @expectedExceptionMessage De status van de boeking moet Concept zijn
+     */
+    public function testDeletionWorks()
```

Comment:
This looks more like `testDeleteThrowsWhenReturnedXmlContainsErrors`.

Can we also test the happy path?